File: server/ui/src/app/api/models/openttd-server-config-get.ts
/* tslint:disable */
/* eslint-disable */
import { OpenttdServer } from './openttd-server';
export interface OpenttdServerConfigGet {
  autoSaveMinutes?: number;
  numberOfAutoSaveFilesToKeep?: number;
  numberOfManuallySaveFilesToKeep?: number;
  servers?: Array<OpenttdServer>;
}

File: src/main/java/de/litexo/model/external/OpenttdServerConfigGet.java
package de.litexo.model.external;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class OpenttdServerConfigGet {
    private int autoSaveMinutes;
    private int numberOfAutoSaveFilesToKeep;
    private int numberOfManuallySaveFilesToKeep;
    List<OpenttdServer> servers = new ArrayList<>();
}
